hadoop-ozone/integration-test-s3/src/test/java/org/apache/hadoop/ozone/s3/awssdk/v1/AbstractS3SDKV1Tests.java

```diff
@@ -532,81 +532,108 @@ public void testGetObjectWithoutETag() throws Exception {
   }
 
   @Test
-  public void testListObjectsMany() {
-    final String bucketName = getBucketName();
-    s3Client.createBucket(bucketName);
-    final List<String> keyNames = Arrays.asList(
-        getKeyName("1"),
-        getKeyName("2"),
-        getKeyName("3")
-    );
-
-    for (String keyName: keyNames) {
-      s3Client.putObject(bucketName, keyName, RandomStringUtils.secure().nextAlphanumeric(5));
-    }
-
-    ListObjectsRequest listObjectsRequest = new ListObjectsRequest()
-        .withBucketName(bucketName)
-        .withMaxKeys(2);
-    ObjectListing listObjectsResponse = s3Client.listObjects(listObjectsRequest);
-    assertThat(listObjectsResponse.getObjectSummaries()).hasSize(2);
-    assertEquals(bucketName, listObjectsResponse.getBucketName());
-    assertEquals(listObjectsResponse.getObjectSummaries().stream()
-        .map(S3ObjectSummary::getKey).collect(Collectors.toList()),
-        keyNames.subList(0, 2));
-    assertTrue(listObjectsResponse.isTruncated());
-
-
-    listObjectsRequest = new ListObjectsRequest()
-        .withBucketName(bucketName)
-        .withMaxKeys(2)
-        .withMarker(listObjectsResponse.getNextMarker());
-    listObjectsResponse = s3Client.listObjects(listObjectsRequest);
-    assertThat(listObjectsResponse.getObjectSummaries()).hasSize(1);
-    assertEquals(bucketName, listObjectsResponse.getBucketName());
-    assertEquals(listObjectsResponse.getObjectSummaries().stream()
-            .map(S3ObjectSummary::getKey).collect(Collectors.toList()),
-        keyNames.subList(2, keyNames.size()));
-    assertFalse(listObjectsResponse.isTruncated());
+  public void testListObjectsMany() throws Exception {
+    testListObjectsMany(false);
   }
 
   @Test
-  public void testListObjectsManyV2() {
+  public void testListObjectsManyV2() throws Exception {
+    testListObjectsMany(true);
+  }
+
+  private void testListObjectsMany(boolean isListV2) throws Exception {
```

Comment:
Why not use @ParameterizedTest to parameterize the test?

Reply:
Thanks for the review. Currently because `getBucketName` uses the test name as part of the bucket name, I'm expecting that the bucket name of both the list v1 and list v2 tests will be the same, causing BUCKET_ALREADY_EXISTS exception.

Generally I usually use parameterized if there are three or more different possible parameters since without parameterized test, it will be more verbose. However, in this case, adding parameterized for a simple boolean might make the test more verbose.

We can improve the parameterization in the future if needed. Let me know what you think.